<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkDICOMAppWidget</class>
 <widget class="QWidget" name="ctkDICOMAppWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>638</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ctkDICOMAppWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="windowTitle">
      <string>toolBar</string>
     </property>
     <property name="floatable">
      <bool>true</bool>
     </property>
     <addaction name="actionImport"/>
     <addaction name="actionExport"/>
     <addaction name="actionQuery"/>
     <addaction name="actionSend"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="topLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>LocalDatabase:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ctkDirectoryButton" name="directoryButton" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="queryLayout" stretch="1,0">
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="searchOptionLayout">
         <item>
          <widget class="ctkDICOMQueryWidget" name="searchOption" native="true"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="userFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="viewerLayout">
       <item>
        <widget class="ctkDICOMThumbnailListWidget" name="thumbnailsWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="previewFrame">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>256</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="prevStudyButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>32</horstretch>
                <verstretch>32</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Previous study</string>
              </property>
              <property name="text">
               <string>&lt;&lt;&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="prevSeriesButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>32</horstretch>
                <verstretch>32</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Previous series</string>
              </property>
              <property name="text">
               <string>&lt;&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="prevImageButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>32</horstretch>
                <verstretch>32</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Previous image</string>
              </property>
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextImageButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>32</horstretch>
                <verstretch>32</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Next image</string>
              </property>
              <property name="text">
               <string>&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextSeriesButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>32</horstretch>
                <verstretch>32</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Next series</string>
              </property>
              <property name="text">
               <string>&gt;&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextStudyButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>32</horstretch>
                <verstretch>32</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Next study</string>
              </property>
              <property name="text">
               <string>&gt;&gt;&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="ctkDICOMDatasetView" name="imagePreview" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="autoPlayCheckbox">
              <property name="text">
               <string>auto-play</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="playSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import a DICOM file or folder</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export selected study/series to a DICOM folder (not yet available)</string>
   </property>
  </action>
  <action name="actionQuery">
   <property name="text">
    <string>Query</string>
   </property>
   <property name="toolTip">
    <string>Query and Retrieve DICOM studies from a DICOM node</string>
   </property>
  </action>
  <action name="actionSend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="toolTip">
    <string>Send DICOM Studies to a DICOM node (not yet available)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDICOMQueryWidget</class>
   <extends>QWidget</extends>
   <header>ctkDICOMQueryWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
   <container>1</container>
   <slots>
    <signal>directoryChanged(QString)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ctkDICOMThumbnailListWidget</class>
   <extends>QWidget</extends>
   <header>ctkDICOMThumbnailListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDICOMDatasetView</class>
   <extends>QWidget</extends>
   <header location="global">ctkDICOMDatasetView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionImport</sender>
   <signal>triggered(bool)</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>openImportDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered(bool)</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>openExportDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuery</sender>
   <signal>triggered(bool)</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>openQueryDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextImageButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>onNextImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevImageButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>onPreviousImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextSeriesButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>onNextSeries()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevSeriesButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>onPreviousSeries()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextStudyButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>onNextStudy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prevStudyButton</sender>
   <signal>clicked()</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>onPreviousStudy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoPlayCheckbox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ctkDICOMAppWidget</receiver>
   <slot>onAutoPlayCheckboxStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>openImportDialog()</slot>
  <slot>openExportDialog()</slot>
  <slot>openQueryDialog()</slot>
  <slot>onDatabaseDirectoryChaged(QString)</slot>
  <slot>onNextImage()</slot>
  <slot>onPreviousImage()</slot>
  <slot>onNextSeries()</slot>
  <slot>onPreviousSeries()</slot>
  <slot>onNextStudy()</slot>
  <slot>onPreviousStudy()</slot>
  <slot>onAutoPlayCheckboxStateChanged(int)</slot>
 </slots>
</ui>
